torch/onnx/symbolic_opset11.py

```diff
@@ -117,3 +117,40 @@ def unique_dim(g, self, dim, sorted, return_inverse, return_counts):
 
 def round(g, self):
     return g.op("Round", self)
+
+
+def arange(g, *args):
+    def _get_arange_dtype(dtype):
+        dtype = sym_help._maybe_get_const(dtype, 'i')
+        if sym_help._is_value(dtype):
+            dtype = 4  # default to int64
+        return dtype
+
+    if len(args) == 5:
+        # aten::arange(Scalar end, ScalarType dtype, Layout, Device, bool pin_memory)
+        dtype = _get_arange_dtype(args[1])
+        start_default = g.op("Constant", value_t=torch.tensor(0, dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
+        delta_default = g.op("Constant", value_t=torch.tensor(1, dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
+        arange_tensor = g.op("Range", start_default, args[0], delta_default)
+    elif len(args) == 6:
+        # aten::arange(Scalar start, Scalar end, ScalarType dtype, Layout, Device, bool pin_memory)
+        dtype = _get_arange_dtype(args[2])
+        start_default = g.op("Constant", value_t=torch.tensor(0, dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
+        delta_default = g.op("Constant", value_t=torch.tensor(1, dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
+        arange_tensor = g.op("Range", args[0], args[1], delta_default)
```

Comment:
here you create start_default and delta_default but never use them.
Same below.

Reply:
Oh right, fixed it. Thanks